### PR TITLE
Dispatch 1.3 GA docs 

### PR DESCRIPTION
**Pull request by d2iq-dispatch**

### CODE REVIEW

**Review by jbarrick-mesosphere — APPROVED** [review]

File: pages/ksphere/dispatch/1.3/references/dispatchfile/index.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ layout: layout.pug
 navigationTitle:  Dispatchfile
 title: Dispatchfile
 menuWeight: 20
-beta: true
+beta: false
```
Comment (cmastr) [inline pages/ksphere/dispatch/1.3/references/dispatchfile/index.md:6]: why do we have an empty topic here?

File: pages/ksphere/dispatch/1.3/tutorials/ci_tutorials/local-runner/index.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ dispatch ci run local --task unit-test-simple
 
 - This command assumes that the git repo (directory containing `.git` folder) exists in the current working directory. You can override it using the `--git-repo` flag.
 - To retain the cluster (for debugging, reuse etc.,) pass `--skip-cluster-delete` flag.
-- To install certain manifests into the cluster (such as installing tiller service or any other bootstrap manifests), `--with-file` flag can be specified.
+- To install certain manifests into the cluster, `--with-file` flag can be specified.
```
Comment (cneth) [inline pages/ksphere/dispatch/1.3/tutorials/ci_tutorials/local-runner/index.md:136]:
```suggestion
- To install certain manifests into the cluster, the `--with-file` flag can be specified.
```

**Review by cneth — CHANGES_REQUESTED** [review]
LGTM, just a few nits here and there.

File: pages/ksphere/dispatch/1.3/references/dispatchfile/index.md (hunk 1 above)
Comment (chhsia0) [reply pages/ksphere/dispatch/1.3/references/dispatchfile/index.md:6]: I'm not sure why we had this but it's probably there for a while. I guess we could remove this.

**Comment by chhsia0** [comment]
@cmastr Please don't merge this just yet. We're going to re-cutting 1.3.0

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
> @cmastr Please don't merge this just yet. We're going to re-cutting 1.3.0

No worries. I'm expecting this to be the GA docs, so won't merge until 27th. Preview looks good!

**Review by cmastr — APPROVED** [review]

**Review by cneth — APPROVED** [review]

**Pull request merged**